pallets/funding/src/functions.rs

```diff
@@ -1309,8 +1332,22 @@ impl<T: Config> Pallet<T> {
 			InvestorType::Professional => project_metadata.contributing_ticket_sizes.professional,
 			InvestorType::Retail => project_metadata.contributing_ticket_sizes.retail,
 		};
+		let max_multiplier = match investor_type {
+			InvestorType::Retail => {
+				RetailParticipations::<T>::mutate(&did, |project_participations| {
+					if project_participations.contains(&project_id).not() {
+						// We don't care if it fails, since it means the user already has access to the max multiplier
+						let _ = project_participations.try_push(project_id);
+					}
+					retail_max_multiplier_for_participations(project_participations.len() as u8)
+				})
+			},
 
+			InvestorType::Professional => PROFESSIONAL_MAX_MULTIPLIER,
+			InvestorType::Institutional => INSTITUTIONAL_MAX_MULTIPLIER,
+		};
 		// * Validity checks *
+		ensure!(multiplier.into() <= max_multiplier && multiplier.into() > 0u8, Error::<T>::ForbiddenMultiplier);
```

Comment:
We have a bug in the do_contribute_(community/remainder)_round functions as we are not limiting the types of investors anymore. We just pass the investor type to the inner function. But Institutional (and professional?) cannot invest in community round.

Reply:
Institutional and professional CAN invest in community round. As long as they have no winning bids